lib/components/Forbidden403/Forbidden403.tsx

```diff
@@ -0,0 +1,21 @@
+import { Section } from '../../layouts/Section/Section'
+import { PageHeader } from '../PageHeader/PageHeader'
+
+export interface Forbidden403Props {
+  children?: React.ReactNode
+  content?: string
+}
+
+export const Forbidden403 = ({
+  children,
+  content = 'You do not have permission to access this',
+}: Forbidden403Props) => {
+  return (
+    <>
+      <Section maxWidth="7xl">
+        <PageHeader as="h1" header="403 - Forbidden" content={content} size="md" isCenter />
+        {children}
+      </Section>
+    </>
```

Comment:
The section should not be used in a single component in this way, all you need if the PageHeader component. You could wrap the component in Section in the stories file, this would apply prose and format spacing properly

Reply:
We should hardcode the children for consistency and maybe just have a isCenter prop to be future proof, with center as default. Here's what this component could look like, taking text from the existing 404 on CMS sites.

```
<>
  <PageHeader
    as="h1"
    header="403 - Forbidden"
    content="Apologies, but you do not have permission to access this page"
    size="lg"
  />
  <p>
    Please use the site navigation located at the top of this page or search the site using the search field icon in the far right side of the navigation.
  </p>
  <ButtonGroup>
    <LinkComponent href="/" className="cu-button cu-button--red">
      Site Homepage
    </LinkComponent>
    <LinkComponent href="https://carleton.ca" className="cu-button cu-button--grey">
      Carleton Homepage
    </LinkComponent>
  </ButtonGroup>
</>
```

Reply:
done